server/sonar-server/src/main/java/org/sonar/server/computation/task/projectanalysis/period/PeriodsHolderImpl.java

```diff
@@ -47,7 +47,9 @@
    * @throws IllegalArgumentException if the specified Iterable has more than 5 elements
    * @throws IllegalStateException if the holder has already been initialized
    * @throws IllegalStateException if two Periods have the same index
+   * @deprecated as only one period is now available. Use {@link #setPeriod(Period)} instead
    */
+  @Deprecated
   public void setPeriods(Iterable<Period> periods) {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133)